edi_endpoint_oca/models/edi_backend.py

```diff
@@ -16,10 +16,16 @@ class EDIBackend(models.Model):
     )
     endpoints_count = fields.Integer(compute="_compute_endpoints_count")
 
-    @api.depends("endpoint_ids")
+    @api.depends("endpoint_ids.active")
     def _compute_endpoints_count(self):
+        data = self.env["edi.endpoint"].read_group(
+            [("backend_id", "in", self.ids), ("active", "=", True)],
```

Comment:
`("active", "=", True)` shouldn't be necessary

Reply:
It's just more explicit.